specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -454,34 +534,49 @@ namespace Microsoft.Windows.PushNotifications
     static runtimeclass PushNotificationManager
     {
         // Register an activator using an ActivationInfo context and return a RegistrationToken
-        static PushNotificationRegistrationToken RegisterActivator(PushNotificationActivationInfo details);
+        static void RegisterActivator(PushNotificationActivationInfo details);
```

Comment:
A little concerned that we removed the token here. The token is being used to infer state like the BackgroundTaskName in the implementation today. How would that flow work without a token?

Reply:
PushNotificationRegistrationToken moving to private struct and we cache generated tokens by the user to revoke when given unregister flag.